### PR TITLE
private/pedro/references menu improvements

### PR DESCRIPTION
**Pull request by pedropintosilva**
- References: Move missing items to compact view menu entry
- References (compact view): Always show bibliography

### CODE REVIEW

**Review by timar — APPROVED** [review]

File: browser/src/control/Control.Menubar.js [hunk 1]
```diff
@@ -1636,6 +1640,8 @@ L.Control.Menubar = L.Control.extend({
 								$(aItem).removeClass(constChecked);
 							}
 						}
+					} else if ((id.startsWith('zotero') && !(self._map.zotero && self._map.zotero.enable))) {
+						$(aItem).addClass('disabled');
```
Comment (pedropintosilva) [inline browser/src/control/Control.Menubar.js:1644]: @timar there was conflicts due to recent references related PR merges. I have rebased. But now the menu entries do not get disabled if zotero is not enabled maybe something from https://github.com/CollaboraOnline/online/pull/5582 impacted this?

Comment (timar) [reply browser/src/control/Control.Menubar.js:1644]: Probably, it's cool that @eszkadev fixed and merged my PR. I'll take care of this PR next, don't worry.

**Comment by timar** [comment]
@pedropintosilva it works for me...
image

**Comment by timar** [comment]
Isn't it possible to hide menu items instead of disabling them?

**Comment by pedropintosilva** [comment]
> Isn't it possible to hide menu items instead of disabling them?

Yes, but that's what I don't want

**Comment by pedropintosilva** [comment]
Ran tests locally, discovered what was the problem. Insert footer and endnote cannot be removed from insert menu because. Added them back in.

**Comment by pedropintosilva** [comment]
just DCO is waiting with no apparent reason, merging.

**Comment by pedropintosilva** [comment]
... need to rebase again, more stuff entered

**Pull request closed without merging**